### ISSUE TITLE
Redmine-Git-Branch: add a "start" option

### ISSUE THREAD

**Issue by mgdesign** [sparkfabrik/sparktool#61]
Using the command
`spark redmine:git:branch ####`
it would be useful an additional option that enables the user to assign to herself the issue and change its status.

Eg.
`spark redmine:git:branch #### --start`
Should:
1) create the branch (current implementation)
2) push -u the branch (current implementation) 
3) move to the brach (current implementation)
4) se the issue #### to status "in progress"
5) assign the issue #### to the user using the command

**Comment by paolomainardi** [comment]
the first 3 points are already available using the `redmine:git:branch` command.

We need to add the "redmine:issue:update" command in order to achieve the 4 and 5 point.

As far as i know @stickgrinder started to work on this, can you confirm ?

**Comment by stickgrinder** [comment]
Yes, I was slowly working on a command to PUT an issue update (issue #76). Since I can't afford full focus on this I will limit myself to update operation so far, so this issue can be worked out.
